pkg/install/stack/kubermatic/validation.go

```diff
@@ -101,11 +103,19 @@ func validateHelmValues(config *operatorv1alpha1.KubermaticConfiguration, helmVa
 	}
 
 	path = yamled.Path{"kubermaticOperator", "imagePullSecret"}
-	if value, _ := helmValues.GetString(path); value == "" {
+	if value, _ := helmValues.GetString(path); value == "" && config.Spec.ImagePullSecret != "" {
 		logger.Warnf("Helm values: %s is empty, setting to spec.imagePullSecret from KubermaticConfiguration", path.String())
 		helmValues.Set(path, config.Spec.ImagePullSecret)
 	}
 
+	repoPath := yamled.Path{"kubermaticOperator", "image", "repository"}
+	repo, _ := helmValues.GetString(repoPath)
```

Comment:
Is it safe to ignore errors?

Reply:
This is not ignoring an error, the second return value is a boolean telling you if the value was found in the document. This is not relevant here, as I'm selecting a string and the empty value will just be an empty string. The 2nd return value is useful if you're getting booleans out of the document, for example.